schemas/Interaction.json

```diff
@@ -44,6 +44,7 @@
         {"const": "WebRoomListHeaderPlusMenuCreateChatItem", "description": "User clicked the create DM button in the + context menu of the room list header in Element Web/Desktop." },
         {"const": "WebSpaceContextMenuHomeItem", "description": "User clicked the home button in the context menu of a space in Element Web/Desktop." },
         {"const": "WebHomeExploreRoomsButton", "description": "User clicked the explore rooms button in the home page of Element Web/Desktop." },
+        {"const": "WebHomeMiniAvatarUploadButton", "description": "User updated their avatar via the mini avatar uploader in the home page of Element Web/Desktop." },
```

Comment:
this description is no longer right

Reply:
Thanks for catching that, I had totally missed it!